t/21-needles.t

```diff
@@ -32,6 +39,15 @@ my %settings = (
     ARCH => 'x86_64',
 );
 
+my $mock_utils = Test::MockModule->new('OpenQA::Utils');
+$mock_utils->redefine('run_cmd_with_log', sub { 1 });
+$mock_utils->redefine(
```

Comment:
I can't really explain why this doesn't appear as covered.
It is actually used, and even if the mocked method wasn't used, at least the `redefine` should be covered.
I checked if this could be a fallout from our coverage workaround, but I see the same result without it...